### PR TITLE
Add slack and stackoverflow icon to footer in docs

### PR DESCRIPTION
**Pull request by gsheni**

### CODE REVIEW

File: docs/source/_templates/layout.html [hunk 1]
```diff
@@ -38,10 +38,16 @@
     </div>
     <div class="footer-cell-2">
       <a href="https://github.com/alteryx/evalml#readme" target="_blank">
-        <img class="footer-image-github" src="{{ pathto('_static/images/github.svg', 1) }}" alt="GitHub">
+        <img  class="footer-image-github" src="{{ pathto('_static/images/github.svg', 1) }}" alt="GitHub">
```
Comment (gsheni) [inline docs/source/_templates/layout.html:41]:
```suggestion
        <img class="footer-image-github" src="{{ pathto('_static/images/github.svg', 1) }}" alt="GitHub">
```

File: docs/source/_templates/layout.html [hunk 2]
```diff
@@ -41,7 +41,13 @@
         <img class="footer-image-github" src="{{ pathto('_static/images/github.svg', 1) }}" alt="GitHub">
       </a>
       <a href="https://twitter.com/AlteryxOSS" target="_blank">
-        <img class="footer-image-twitter" src="{{ pathto('_static/images/twitter.svg', 1) }}" alt="Twitter">
+        <img  class="footer-image-twitter" src="{{ pathto('_static/images/twitter.svg', 1) }}" alt="Twitter">
```
Comment (gsheni) [inline docs/source/_templates/layout.html:44]:
```suggestion
        <img class="footer-image-twitter" src="{{ pathto('_static/images/twitter.svg', 1) }}" alt="Twitter">
```

**Comment by freddyaboulton** [comment]
For some reason the change is not displaying in RTD

image

**Comment by gsheni** [comment]
@freddyaboulton I am fairly certain this will work. Both Featuretools and Woodwork use this same exact code and it renders properly in the footer:
- https://woodwork.alteryx.com/en/latest/
- https://featuretools.alteryx.com/en/latest/

**Comment by freddyaboulton** [comment]
Looks good except that the stack overflow icon does not render:

image

File: docs/source/_templates/layout.html [hunk 3]
```diff
@@ -43,6 +43,12 @@
       <a href="https://twitter.com/AlteryxOSS" target="_blank">
         <img class="footer-image-twitter" src="{{ pathto('_static/images/twitter.svg', 1) }}" alt="Twitter">
       </a>
+      <a href="https://join.slack.com/t/alteryx-oss/shared_invite/zt-182tyvuxv-NzIn6eiCEf8TBziuKp0bNA" target="_blank">
+        <img  class="footer-image-github" src="{{ pathto('_static/images/slack.svg', 1) }}" alt="Slack">
```
Comment (gsheni) [inline docs/source/_templates/layout.html:47]:
```suggestion
        <img  class="footer-image-slack" src="{{ pathto('_static/images/slack.svg', 1) }}" alt="Slack">
```
```suggestion
        <img  class="footer-image-github" src="{{ pathto('_static/images/slack.svg', 1) }}" alt="Slack">
```

Comment (gsheni) [inline docs/source/_templates/layout.html:47]:
```suggestion
        <img  class="footer-image-slack" src="{{ pathto('_static/images/slack.svg', 1) }}" alt="Slack">
```
```suggestion
        <img  class="footer-image-github" src="{{ pathto('_static/images/slack.svg', 1) }}" alt="Slack">
```

File: docs/source/_templates/layout.html [hunk 4]
```diff
@@ -43,6 +43,12 @@
       <a href="https://twitter.com/AlteryxOSS" target="_blank">
         <img class="footer-image-twitter" src="{{ pathto('_static/images/twitter.svg', 1) }}" alt="Twitter">
       </a>
+      <a href="https://join.slack.com/t/alteryx-oss/shared_invite/zt-182tyvuxv-NzIn6eiCEf8TBziuKp0bNA" target="_blank">
+        <img  class="footer-image-slack" src="{{ pathto('_static/images/slack.svg', 1) }}" alt="Slack">
+      </a>
+      <a href="https://stackoverflow.com/questions/tagged/evalml" target="_blank">
+        <img  class="footer-image-github" src="{{ pathto('_static/images/stackoverflow.svg', 1) }}" alt="Stack Overflow">
```
Comment (gsheni) [inline docs/source/_templates/layout.html:50]:
```suggestion
        <img  class="footer-image-stack-overflow" src="{{ pathto('_static/images/stackoverflow.svg', 1) }}" alt="Stack Overflow">
```

File: docs/source/conf.py [hunk 5]
```diff
@@ -106,8 +106,32 @@
...
+            "name": "Slack",
+            "url": "https://join.slack.com/t/alteryx-oss/shared_invite/zt-182tyvuxv-NzIn6eiCEf8TBziuKp0bNA",
+            "icon": "fab fa-slack",
+            "type": "fontawesome",
+        },
+        {
+            "name": "StackOverflow",
+            "url": "https://stackoverflow.com/questions/tagged/evalml",
+            "icon": "fab fa-stack-overflow",
+            "type": "fontawesome'",
```
Comment (gsheni) [inline docs/source/conf.py:132]:
```suggestion
            "type": "fontawesome",
```

**Comment by gsheni** [comment]
@freddyaboulton solved:
https://github.com/alteryx/evalml/pull/3528/commits/d1a2cbecfc01a26e99658b2183138a3d1a5a7a12

**Comment by gsheni** [comment]
Screen Shot 2022-05-24 at 1 25 07 PM
Works now

**Review by freddyaboulton — APPROVED** [review]
Thanks @gsheni !

**Pull request merged**